docs/utilities/truncate.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+---
+
+## Truncation
+
+<hr>
+
+To truncate text, use the class `u-truncate`.
+
+When applied to a parent container, it will truncate any elements placed inside, so long as the parent's width is set.
+When applied to individual elements within a parent, e.g. a table cell, only those elements will be truncated.
```

Comment:
Maybe warn that this will hide tooltips and if you want tooltips you should truncate inside the tooltip.

Reply:
I've added a note of warning. I've also experimented a bit and I don't think placing a tooltip inside the truncated element can work. The only way I can think of would be to use js to position an independent tooltip to coincide with the truncate element's position.